sympy/physics/tests/test_units.py

```diff
@@ -20,6 +20,8 @@ def test_units():
     t = Symbol('t')
     assert integrate(t*m/s, (t, 1*s, 5*s)) == 12*m*s
     assert (t * m/s).integrate((t, 1*s, 5*s)) == 12*m*s
+    
+    abs((year / week).evalf() - 52) < 0.2
```

Comment:
There is no `assert` statement and why don't you use the "exact" value and a tighter tolerance? Or you can `evalf` to the exact number of digits you want and compare to a sympified float.